bug-finding.ipynb

```diff
@@ -290,11 +355,18 @@
    "source": [
     "# Bug 3 analysis and response"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I'm not entirely sure how to fix it. For the old code, I think the problem was with the window. The original code calculated all windows at the same time and it's not specific to each collocate pairs. "
```

Comment:
Sort of. The old code was double-counting pairs by that were close together within the window. Your solution fixes it, but it would be great to see a more complete explanation of _why_ it fixes the problem.